### PR TITLE
[FLINK-4623] Create Physical Execution Plan of a DataStream

### PR DESCRIPTION
**Pull request by tonycox**

### CODE REVIEW

File: flink-libraries/flink-table/src/test/scala/resources/testUnionStream0.out [hunk 1]
```diff
@@ -2,3 +2,21 @@
...
+
+	Stage 3 : Operator
+		content : from: (count, word)
+		ship_strategy : REBALANCE
+		exchange_mode : null
+
+		Stage 4 : Operator
+			content : from: (count, word)
+			ship_strategy : REBALANCE
+			exchange_mode : null
```
Comment (wuchong) [inline flink-libraries/flink-table/src/test/scala/resources/testUnionStream0.out]: It seems that the json execution plan generated from `env.getExecutionPlan` dosen't contain the `exchange_mode` information. Could we remove this node in explain?

File: flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/StreamTableEnvironment.scala [hunk 2]
```diff
@@ -311,14 +313,24 @@ abstract class StreamTableEnvironment(
...
     */
-   def explain(table: Table): String = {
+  def explain(table: Table): String = {
 
     val ast = RelOptUtil.toString(table.getRelNode)
 
+    val dataStream = translate[Row](table)(TypeExtractor.createTypeInfo(classOf[Row]))
+
+    val env = dataStream.getExecutionEnvironment
+    val jasonSqlPlan = env.getExecutionPlan
```
Comment (fhueske) [inline flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/StreamTableEnvironment.scala]: `jason` -> `json`

File: flink-libraries/flink-table/src/test/scala/org/apache/flink/api/scala/stream/ExplainStreamTest.scala [hunk 3]
```diff
@@ -40,9 +40,12 @@ class ExplainStreamTest
       .toTable(tEnv, 'a, 'b)
       .filter("a % 2 = 0")
 
-    val result = tEnv.explain(table).replaceAll("\\r\\n", "\n")
+    val result = tEnv.explain(table)
+      .replaceAll("\\r\\n", "\n").replaceAll("Stage \\d+", "")
```
Comment (fhueske) [inline flink-libraries/flink-table/src/test/scala/org/apache/flink/api/scala/stream/ExplainStreamTest.scala]: Please add a comment why you ignore the `Stage` info in the test.

**Comment by fhueske** [comment]
+1 to merge.

Btw @tonycox, please leave a brief comment when you update a PR. 
Otherwise, we won't get an email and the update might not be noticed until somebody looks at the PR. Thanks!

**Comment by fhueske** [comment]
Merging :-)

**Pull request closed without merging**